### DISCUSSION TITLE
Detailed Instructions for Dummies 🫖

### DISCUSSION THREAD

**Discussion by netn992** [shadowsocks/shadowsocks-rust#1063] [Q&A]
Hi developers! Please create a detailed step by step (1. command 2. command 3. command etc) instructions to start the server for ordinary people, so many times have tried to run and all without success. As well as detailed instructions on how to set up a configuration file. Thank you very much for your work!

**Comment by fatemiinfo** [comment]
For whom it may concern, We really need it for Iran, Specified config needed:
1-Being able to change port number
2-Authentication
3-We need it to be easy to use for mobile and pc users.
Its a War. Please help us
Thank you for developing Shadowsocks

**Comment by mega-optimus** [comment]
This is a good introduction.
https://shadowsocks.org/guide/

You can also search online for the keywords "Shadowsocks" "VPS"

**Reply by b9Joker108** [reply]
I noticed that the guide doesn't exist anymore, is it being edited?
